### PR TITLE
dcp_stateassemblydistricts and dcp_stateassemblydistricts_wi created

### PR DESCRIPTION
**Pull request by rubenvecg**
Subissue #88 
Issue #58

### CODE REVIEW

File: library/templates/dcp_stateassemblydistricts_wi.yml [hunk 1]
```diff
@@ -0,0 +1,36 @@
+dataset:
+  name: &name dcp_stateassemblydistricts
```
Comment (mgraber) [inline library/templates/dcp_stateassemblydistricts_wi.yml:2]: Should be `dcp_stateassemblydistricts_wi`

File: library/templates/dcp_stateassemblydistricts_wi.yml [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+  version: "{{ version }}"
+  acl: public-read
+
+  source:
+    url:
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyadwi_{{ version }}.zip
+      subpath: nyadwi_{{ version }}/nyadwi.shp
+    geometry:
+      SRS: EPSG:4326
```
Comment (mgraber) [inline library/templates/dcp_stateassemblydistricts_wi.yml:11]: Should be `EPSG:2263` now that this is getting pulled from Bytes.

File: library/templates/dcp_stateassemblydistricts.yml [hunk 3]
```diff
@@ -0,0 +1,36 @@
...
+  name: &name dcp_stateassemblydistricts
+  version: "{{ version }}"
+  acl: public-read
+
+  source:
+    url:
+      path: https://www1.nyc.gov/assets/planning/download/zip/data-maps/open-data/nyad_{{ version }}.zip
+      subpath: nyad_{{ version }}/nyad.shp
+    geometry:
+      SRS: EPSG:4326
```
Comment (mgraber) [inline library/templates/dcp_stateassemblydistricts.yml:11]: Should be `EPSG:2263` now that this is getting pulled from Bytes.

File: library/templates/dcp_stateassemblydistricts.yml [hunk 4]
```diff
@@ -0,0 +1,36 @@
...
+      - "PRECISION=NO"
+    fields: []
+    sql: null
+
+  info:
+    description: |
+      ### ESRI Shapefile
+      New York State Assembly District boundaries for the City of New York (Clipped to shoreline). 
+      These district boundaries represent the redistricting as of the US Census 2010.
+      Source will be updated once the 2020 Census data is available.
```
Comment (mgraber) [inline library/templates/dcp_stateassemblydistricts.yml:33]: Maybe "Source will be updated once the redistricted in response to the 2020 Census"

**Review by mgraber — CHANGES_REQUESTED** [review]

File: library/templates/dcp_stateassemblydistricts.yml (hunk 4 above)
Comment (rubenvecg) [reply library/templates/dcp_stateassemblydistricts.yml:33]: Or maybe "...once the district boundaries get redistricted..."?

**Pull request merged**